csharp/src/Microsoft.ML.OnnxRuntime/OrtValue.shared.cs

```diff
@@ -889,14 +932,196 @@ public static OrtValue CreateMap(OrtValue keys, OrtValue values)
         {
             if (keys is null || values is null)
             {
-                throw new ArgumentNullException($"keys or/and values are null");
+                throw new ArgumentNullException("keys or/and values are null");
             }
 
+            var compositeMembers = new DisposableList<OrtValue>
+            {
+                keys,
+                values
```

Comment:
It's not clear if this is creating a map with a single key/value or not. Are the `keys` and `values` inputs sequences or single values? Doco should clarify what's supported/expected.

Reply:
Will clarify the documentation.